src/Features/CSharp/Portable/Structure/CSharpBlockStructureProvider.cs

```diff
@@ -40,7 +40,7 @@ internal class CSharpBlockStructureProvider : AbstractBlockStructureProvider
             builder.Add<RegionDirectiveTriviaSyntax, RegionDirectiveStructureProvider, MetadataAsSource.MetadataRegionDirectiveStructureProvider>();
             builder.Add<SimpleLambdaExpressionSyntax, SimpleLambdaExpressionStructureProvider>();
             builder.Add<StructDeclarationSyntax, TypeDeclarationStructureProvider, MetadataAsSource.MetadataTypeDeclarationStructureProvider>();
-            builder.Add<SwitchStatementSyntax, SwitchStatementStructureProvider>();
+            builder.Add(typeof(SwitchStatementSyntax), ImmutableArray.Create<AbstractSyntaxStructureProvider>(new SwitchStatementStructureProvider(includeInternalStructures: true)));
```

Comment:
please don't do this.  the provider shouldn'tbe configurable.  especially since the configuration is used only for testing purposes.